### PR TITLE
Add initial scaffolding for ETL pipeline

### PR DESCRIPTION
**Pull request by skeshive**
This cloud function is not yet deployed to GCP but will be used for transforming/exporting both flood and atmospheric model predictions to ClimaSens. 

1. export_model_predictions() will get triggered whenever a new object gets added to the climateiq-predictions bucket
2. The new object will be named like "prediction-type/model-id/study-area-name/scenario-id/chunk-id" so we can fetch metadata from the metastore

### CODE REVIEW

File: cloud_functions/climateiq_data_export_cf/main.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# Triggered by the "object finalized" Cloud Storage event type.
+@functions_framework.cloud_event
+def export_model_predictions(cloud_event: CloudEvent) -> Optional[dict]:
+    """This function is triggered when a new object is created or an existing
+    object is overwritten in the "climateiq-predictions" storage bucket.
+
+    Args:
+        cloud_event: The CloudEvent representing the storage event.
+    Returns:
+        A dictionary containing the study area metadata or None if the study
```
Comment (sylmak) [inline cloud_functions/climateiq_data_export_cf/main.py]: This actually raises an error, right?

File: cloud_functions/climateiq_data_export_cf/main.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import functions_framework
+import pathlib
+
+from cloudevents.http import CloudEvent
+from google.cloud import firestore
+from typing import Optional
+
+STUDY_AREAS = "study_areas"
```
Comment (sylmak) [inline cloud_functions/climateiq_data_export_cf/main.py]: Nit: Would be clearer to call this STUDY_AREAS_ID or STUDY_AREAS_COLLECTION_ID.

**Review by sylmak — CHANGES_REQUESTED** [review]

File: cloud_functions/climateiq_data_export_cf/main.py (hunk 1 above)
Comment (skeshive) [reply cloud_functions/climateiq_data_export_cf/main.py]: Good catch, updated the comments

File: cloud_functions/climateiq_data_export_cf/main.py (hunk 2 above)
Comment (skeshive) [reply cloud_functions/climateiq_data_export_cf/main.py]: Done

**Review by sylmak — APPROVED** [review]

**Pull request merged**